libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/RestCriConfig.java

```diff
@@ -12,8 +12,16 @@
 @Getter
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
-@JsonIgnoreProperties(ignoreUnknown = false)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class RestCriConfig extends CriConfig {
+    private static final long DEFAULT_REQUEST_TIMEOUT = 30;
+
+    private long requestTimeout;
     private URI credentialUrl;
     private boolean requiresApiKey;
+
+    public long getRequestTimeout() {
+        // This it to avoid having to define the request timeout in config if not desired
+        return requestTimeout == 0 ? DEFAULT_REQUEST_TIMEOUT : requestTimeout;
```

Comment:
If requestTimeout is not defined in the config json, the default value for a long is 0, which is an invalid argument for a timeout, so we have to override to something better. We need to handle this as we can't update the config first due to not ignoring unknown values.
Doing this is actually quite useful though as it allows the value to be optional in the config json.

Reply:
You might want to consider making the field a `Long` and handling `null`, rather than 0? I think that conveys intent more clearly...

It would also allow you to make `null` => 'no timeout', rather than a default, should that be useful.

Reply:
Yep, that makes sense. I'll leave the default as a 30 second timeout I think, unless you think that having no timeout would be preferable?

Reply:
It probably doesn't make much difference tbh - our default Lambda timeout is 40s, but API GW will time out after 29s (and arguably we should reduce both of those - nobody is going to wait that long for a page to load!)

The only benefit of having no timeout is then you don't have to argue about what the default is :D